Yoast/Yoast-theme-public
# makes the search icon clickable and focusses the inputfield

Fixes https://github.com/Yoast/yoast.com/issues/1317

**How to test:**
If you click on the search-icon in the header. the search box should open up and get the focus so you can start typing :)
